spec/spec_helper.rb

```diff
@@ -0,0 +1,8 @@
+# encoding: utf-8
+
+require 'aws-sdk'
+require 'logstash/devutils/rspec/spec_helper'
+require 'logstash/outputs/sqs'
+
+require 'logstash/logging/logger'
```

Comment:
I'm not a Ruby developer, but isn't it a good practice to `require` dependencies explicitly? Otherwise the `spec_helper` will fail if `logstash/logging/logger` is ever removed from `logstash/devutils/rspec/spec_helper`.